integrations/gitlab-v2/clients/gitlab_client.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
+from functools import partial
+from typing import Any, AsyncIterator, Callable, Optional
+
+from loguru import logger
+from port_ocean.utils.async_iterators import (
+    semaphore_async_iterator,
+    stream_async_iterators_tasks,
+)
+
+from .rest_client import RestClient
+
+
+class GitLabClient:
+    DEFAULT_MIN_ACCESS_LEVEL = 10
+    DEFAULT_PARAMS = {
+        "min_access_level": DEFAULT_MIN_ACCESS_LEVEL,
+        "all_available": True,
+    }
```

Comment:
what is the default min access level? how does this effect the customer ?  will he want / need to adjust it ?

is it required? will it support also users with lower permissions than owner?

Reply:
lets add documentation for it then